docs/ubit/accelerometer.md

```diff
@@ -34,6 +37,12 @@ If there is no scheduler running, the values are synchronously read on `get[X,Y,
 calls. Additionally, if you would like to drive accelerometer updates manually `updateSample()`
 can be used.
 
+### Device initialisation
+When the accelerometer object is created it attempts to detect which accelerometer is on board. 
```

Comment:
I think we should show an example of initialisation as that's now a bit different to other things...

It's more up to the user to make this autodectection happening by the way they initialise the accelerometer:
https://github.com/lancaster-university/microbit/blob/master/source/MicroBit.cpp#L71